lib/softoken/pkcs11c.c

```diff
@@ -1985,6 +1986,56 @@ sftk_doHMACInit(SFTKSessionContext *context, HASH_HashType hash,
     return CKR_OK;
 }
 
+/*
+ * common CMAC initialization routine
+ */
+static CK_RV
+sftk_doAESCMACInit(SFTKSessionContext *context,
```

Comment:
Like the Suggestion for to change CMAC_AESInit to CMAC_Init, this should be just CMAC and take a parameter for the mechanism type. Probably the same one you pass to CMAC_Init, and should be exported from freebl.